dev-docs/bidders/trustedstack.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: bidder
+title: TrustedStack
+description: Prebid TrustedStack Bidder Adaptor
+biddercode: trustedstack
+aliasCode : medianet
+gdpr_supported: true
```

Comment:
```suggestion
tcfeu_supported: true
```

This is set to true as the alias adapter (medianet) does support TCF-EU and has a `gvl_id`